spec/models/item_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Item, type: :model do
+  it "is valid with name, description, price" do
+    item = Item.create(name: 'Canon 7d',
+                       description: 'The best camera ever (NOT)',
+                       price: 15)
+
+    expect(item).to be_valid
+  end
+
+  it 'is invalid without a name' do
+    item = Item.create(description: 'The best camera ever (NOT)',
+                       price: 15)
+
+    expect(item).to_not be_valid
+  end
+
+  it 'is invalid without a description' do
+    item = Item.create(name: 'Canon 7d',
+                       price: 15)
+
+    expect(item).to_not be_valid
+  end
+
+  it 'is invalid without a price' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.